Update PROPOSTA.md

migmoroni opened the pull request:
- Melhoria geral na legibilidade e organização do projeto, utilizando de capítulos e generalizações.
- Adicionado mais profissões na área de jogos permitidas ao MEI.
- Aumento de multa (de 500 a 5 mil) Ufir, por não cumprimento de alteração de classificação etária.
- Pequenas simplificações

migmoroni committed:
Update PROPOSTA.md

- Melhoria geral na legibilidade e organização do projeto, utilizando de capítulos e generalizações.
- Adicionado mais profissões na área de jogos.
- Pequenas simplificações

tolstenko commented on PROPOSTA.md:
```diff
@@ -8,6 +8,12 @@
     5. Responda a possiveis pedidos de melhoria;
 
+
+## Para aprovação de mudanças nesta proposta, seguimos as seguintes normas:
```
move essa parte pro arquivo https://github.com/CoPaJE/CoPaJE/blob/main/CONTRIBUTING.md

migmoroni replied:
Retirado esta parte do texto.

migmoroni committed:
Update PROPOSTA.md

- Retirado o link para normas.

migmoroni committed:
Merge branch 'main' into main

migmoroni committed:
Update PROPOSTA.md

Adicionado mais Profissões

migmoroni committed:
Update CONTRIBUTING.md

Adicionado as primeiras regras para apreciação e inicio da estrutura formal do movimento CoPaJE.

migmoroni committed:
Update CONTRIBUTING.md

Pequenas alterações

tolstenko commented:
LGTM. 

Extras:

Se vc nao fizer isso ou se outra pessoa nao se pronunciar como algo bloqueante, tah td bem por mim e eu farei as seguntes mudancas:

Designer eh um termo amplo e seria interessante deixar isso future-proof. Todos as seguintes posicoes sao Game designers. Entao seria interessate categorizar em termo amplo e depois especificar.
- Game designer eh o generalista dentro os designers que faz de tudo um pouco. A definicao dele deveria ser a tipificacao padrao.
- User Interface, User Experience - Nunca unam os dois pq isso vai dar problema. Sao coisos completamente diferentes. UI cria interface, UX cria a experiencia da navegacao. Eh a msm coisa que vc pegar o cara que faz modelo 3d e colocar como o msm cara que faz animacao do modelo 3d. Podem ser a msm pessoa, mas geralmente em AAA nao sao.
- Narrative Designer ou Roteirista. Ele eh designer tbm.
- Sound Designer - Eh o cara que cola e faz funcionar os audios, muscicas, efeitos sonoros etc.
- Temos tambem varios outros tipos de designers tais como economia, gampplay

Sobre developer, eu diria que ele eh o generalista de diversas outras areas. Um developer ou desenvolvedor eh o cara que programa e escreve codigo. Dentro dele, tambem entraria arquitetos de software e sistemas. Aqui vai uma lista simples so p vc ver que se vc quebrar em seguranca, vc tem q quebrar em outras areas tbm. Networking, Seguranca, Gameplay, Editor, Tools, Platform... e uma infinidade de outros

Acho q vale a pena tbm quebrar em marketing, tbm pq marketing de games eh extremamente especifico e complexo. Eh totalmente diferente de fazer marketing de produtos fisico, de app, e tbm qq outra area. Marketing de games eh um monstro por si so.

Favor assistir os creditos de diablo iv para ter mais inspiracao sobre como criar generalizacao de grupos e como especializar classes de trabalho. https://www.youtube.com/watch?v=yDyOcF9Iel8

Germano123 commented:
LGTM.

tolstenko commented:
Aprovado com ressalvas

migmoroni commented:
@tolstenko No caso do Designer concordo com você. Tanto que especifiquei isso no inciso:
- IX - Designer de jogos: a pessoa física que é responsável por projetar a jogabilidade, as mecânicas, os sistemas do jogo, os desafios, as regras, as fases e as interações que compõem a experiência, para jogos de entretenimento. Atua em um escopo amplo, permitindo cumprir todo ou parcial, o papel informado nos incisos II, III, IV, V e VI deste artigo.

Olhe ao final: Atua em um escopo amplo, permitindo cumprir todo ou parcial, o papel informado nos incisos II, III, IV, V e VI deste artigo.

Assim, especifiquei as profissões separadas, mas também, especifiquei a profissão geral, muito usada por pequenos desenvolvedores indie, que fazem um pouco de tudo e que é especificada no pedido dessa profissão a lista de MEI.

Quanto ao UI e UX, tem razao, vou separar numa proxima atualização.

A questao de deixar tudo tão espeficado é a tentativa de deixar claro para empresas AAA, mas tambem a realidade dos indies.
